api/types/payloads.go

```diff
@@ -52,3 +52,10 @@ type SockVariantDTO struct {
 	Price    float64 `json:"price" validate:"required,gt=0,lt=101"`
 	Quantity int     `json:"quantity" validate:"required,gte=0"`
 }
+
+type UpdateAddressRequest struct {
+    Street   string `json:"street" validate:"required,max=100"`         
+    AptUnit  string `json:"apt_unit"`                                   
```

Comment:
nit: make it "aptUnit"

Reply:
I get this error:
<img width="967" alt="image" src="https://github.com/user-attachments/assets/85a1150a-7d91-4faa-b0f4-5a5f9b0079f1">
I'm reading that it needs to start with uppercase for it to work

Reply:
sorry the `json` part like this `json:"aptUnit"` just to be consistent with the rest of the code

Reply:
Fixed! Should we not be writing it how its listed in the database schema file?

Reply:
Typically, JSON you always use the camelCase since that's the standard notation used JavaScript/TypeScript